packages/activation-notice/src/app/components/step1.tsx

```diff
@@ -87,13 +91,7 @@ function Step1() {
         target="_blank"
         rel="noreferrer"
       >
-        <Image
-          name="tips.png"
-          name2x="tips-2x.png"
-          width={150}
-          height={245}
-          $marginTop={60}
```

Comment:
Are we getting rid of the margin top here?

Reply:
Getting rid of this prop because it wasn‘t actually used anywhere and does nothing.